src/Symfony/Component/Security/Http/Attribute/IsGranted.php

```diff
@@ -14,6 +14,10 @@
 use Symfony\Component\ExpressionLanguage\Expression;
 
 /**
+ * Checks if user has permission to access to some resource using security roles.
```

Comment:
This attribute is actually not limited to roles. You can check any permission that you've registered voters for.

Reply:
I updated the description to mention voters 🙂